provider/src/supportedNetworks.ts

```diff
@@ -0,0 +1,31 @@
+import { Env } from './types'
+
+// This is the list of networks currently supported
+const supportedNetworks = (env: Env, networkId: string) => {
```

Comment:
Moving to env vars (already configured in Cloudflare) to avoid exposing them publicly.

Reply:
Wouldn't it be easier to get rid of the switch statement and simply use object then return object[id]?

Reply:
Ha, we need to pass `env` though :/

Reply:
That would be fine, right?

```ts
const object = { 
  "1": env.MAINNET, 
  "5": env.GOERLI 
}

return object[id]
```

inside the function 

It's just a little easier to read than a switch case maze.

Reply:
Ha! Sorry i had misunderstood. Yes. That works too. I'll fix it.